lib/flutter_tts_web.dart

```diff
@@ -125,58 +110,58 @@ class FlutterTtsPlugin {
     }
   }
 
-  void _speak(String text) {
+  void _speak(String? text) {
     if (ttsState == TtsState.stopped || ttsState == TtsState.paused) {
       utterance.text = text;
       if (ttsState == TtsState.paused) {
-        synth.resume();
+        synth!.resume();
       } else {
-        synth.speak(utterance);
+        synth!.speak(utterance);
       }
     }
   }
 
   void _stop() {
     if (ttsState != TtsState.stopped) {
-      synth.cancel();
+      synth!.cancel();
     }
   }
 
   void _pause() {
     if (ttsState == TtsState.playing || ttsState == TtsState.continued) {
-      synth.pause();
+      synth!.pause();
     }
   }
 
   void _setRate(num rate) => utterance.rate = rate * 2.0;
-  void _setVolume(num volume) => utterance.volume = volume;
-  void _setPitch(num pitch) => utterance.pitch = pitch;
-  void _setLanguage(String language) => utterance.lang = language;
+  void _setVolume(num? volume) => utterance.volume = volume;
+  void _setPitch(num? pitch) => utterance.pitch = pitch;
+  void _setLanguage(String? language) => utterance.lang = language;
 
-  bool _isLanguageAvailable(String language) {
+  bool _isLanguageAvailable(String? language) {
     if (voices?.isEmpty ?? true) _setVoices();
     if (languages?.isEmpty ?? true) _setLanguages();
-    for (var lang in languages) {
-      if (lang.toLowerCase() == language.toLowerCase()) return true;
+    for (var lang in languages!) {
+      if (lang!.toLowerCase() == language!.toLowerCase()) return true;
```

Comment:
The use of `lang!` here would suggest that the element type of the list probably should be non-nullable.